rpc/models.py

```diff
@@ -826,3 +826,34 @@ def __str__(self):
             self.by,
             self.time.strftime("%Y-%m-%d"),
         )
+
+
+class SubseriesMember(models.Model):
+    """Tracks which RFC belongs to which subseries and its number"""
+
+    rfc_to_be = models.ForeignKey(RfcToBe, on_delete=models.PROTECT)
+    type = models.ForeignKey("SubseriesType", on_delete=models.PROTECT)
+    number = models.PositiveIntegerField()
+    history = HistoricalRecords()
```

Comment:
Will this history be useful? It will be somewhat rare for a `SubseriesMember` to be created when an RFC is put in a subseries, then deleted if it is taken out. What I think we'd want from history is from the other side of the relationship: "over time, what RFCs has this subseries document comprised?"

Or can we do this efficiently by searching the historical model and I've forgotten?